src/lectures.html

```diff
@@ -0,0 +1,58 @@
+---
+---
+
+<!DOCTYPE html>
+<html lang="pt">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Saira&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="/assets/css/styles.css" />
+    <link rel="shortcut icon" href="/assets/images/favicon.ico" />
+    <title>Palestras do IMEsec</title>
+  </head>
+  <body>
```

Comment:
Wait. All of this is a repetition of `src/index.html`. They could share a template between them

Reply:
Hm, ok! I learned now how I could do that